scaletest/workspacetraffic/metrics.go

```diff
@@ -0,0 +1,49 @@
+package workspacetraffic
+
+import "github.com/prometheus/client_golang/prometheus"
+
+type Metrics struct {
+	BytesRead           prometheus.CounterVec
+	BytesWritten        prometheus.CounterVec
+	Errors              prometheus.CounterVec
+	ReadLatencySeconds  prometheus.HistogramVec
+	WriteLatencySeconds prometheus.HistogramVec
+	LabelNames          []string
+}
+
+func NewMetrics(reg prometheus.Registerer, labelNames ...string) *Metrics {
+	m := &Metrics{
+		BytesRead: *prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: "coderd",
+			Subsystem: "scaletest",
+			Name:      "bytes_read",
+		}, labelNames),
+		BytesWritten: *prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: "coderd",
+			Subsystem: "scaletest",
+			Name:      "bytes_written",
+		}, labelNames),
+		Errors: *prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
Is it used somewhere? Maybe I missed it while reviewing.

Reply:
Nope, I missed actually counting errors 😓 good catch!